### ISSUE TITLE
givens does not work with integer vectors

### ISSUE THREAD

**Issue by hurak** [JuliaLang/julia#32388]
The function `givens` (Julia v1.1.1) does not work properly with vectors with integer arguments. An example code is below:
```
using LinearAlgebra
x = [3,4]
G,r = givens(x,1,2)
```
it returns
```
ERROR: MethodError: no method matching givensAlgorithm(::Int64, ::Int64)
Stacktrace:
 [1] givens(::Int64, ::Int64, ::Int64, ::Int64) at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.1/LinearAlgebra/src/givens.jl:276
 [2] givens(::Array{Int64,1}, ::Int64, ::Int64) at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.1/LinearAlgebra/src/givens.jl:317
 [3] top-level scope at none:0
```

If instead the vector `x` contains floats, there are no problems:
```
x = [3.0,4.0]
G,r = givens(x,1,2)
```
yielding
```
(LinearAlgebra.Givens{Float64}(1, 2, 0.6, 0.8), 5.0)
```

**Comment by andreasnoack** [comment]
This was fixed in https://github.com/JuliaLang/julia/pull/31853

**Comment by hurak** [comment]
> This was fixed in #31853

Great to hear this but how do I access the fixed solution? Sorry for my ignorance of a newcomer to Julia but from the referenced discussion thread I was not quite able to decipher if and how it is possible to get the fixed code. I am on Julia v1.1.1. Shall I go for some nightly build of Julia?
